beaker/application.py

```diff
@@ -187,8 +187,14 @@ def __init__(
                             )
                         action.action.subroutine.implementation = bound_attr
 
+                    assert oc != "clear_state"
                     self.bare_externals[oc] = action
 
+            # Clear state
+            elif handler_config.clear_state is not None:
+                handler_config.clear_state.subroutine.implementation = bound_attr
+                self.bare_clear_state = static_attr
```

Comment:
Is there anything that needs to be checked here? Especially in relation to the above condition